literalai/api/__init__.py

```diff
@@ -212,7 +265,7 @@ def raise_error(error):
                 self.graphql_endpoint,
                 json={"query": query, "variables": variables},
                 headers=self.headers,
-                timeout=10,
+                timeout=timeout,
```

Comment:
I don't see the modification for the async version of `make_gql_call`

Reply:
It is line `1519` and `1532`